R/derive_vars_aage.R

```diff
@@ -64,12 +64,12 @@
 derive_vars_aage <- function(dataset,
                              start_date = BRTHDT,
                              end_date = RANDDT,
-                             unit = "years") {
+                             age_unit = "years") {
   start_date <- assert_symbol(enexpr(start_date))
   end_date <- assert_symbol(enexpr(end_date))
   assert_data_frame(dataset, required_vars = expr_c(start_date, end_date))
   assert_character_scalar(
-    unit,
+    age_unit,
```

Comment:
Hey @manciniedoardo, we need to follow the depreciation strategy for arguments.    https://pharmaverse.github.io/admiraldev/main/articles/programming_strategy.html#argument